clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileStatus.java

```diff
@@ -30,7 +30,7 @@ public String getPhysicalAddress() {
     }
 
     public static class Builder {
-        private Path path;
+        private final Path path;
```

Comment:
Why just this one final field?

Reply:
guess this one is the one that the builder can't change.